stream-chat-android-ui-components/src/main/res/values-en/strings_message_composer.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Translations should be common, but I'd like to unify them (also `MessageInputState`) once we merge this component (or have it ready to merge to develop) to avoid merge conflicts hell.

Reply:
Yeah understandable - we should reuse translations but at this point it's a bit hard with all the conflicts